components/bintray-publish/bin/publish-studio.sh

```diff
@@ -132,18 +131,19 @@ _find_system_commands() {
 info() {
   case "${TERM:-}" in
     *term | xterm-* | rxvt | screen | screen-*)
-      printf -- "   \033[1;36m${program:-unknown}: \033[1;37m${1:-}\033[0m\n"
+      printf -- "   \033[1;36m%s: \033[1;37m%s\033[0m\n" "${program:-unknown}" "${1:-}"
       ;;
     *)
-      printf -- "   ${program:-unknown}: ${1:-}\n"
+      printf -- "   %s: %s\n" "${program:-unknown}" "${1:-}"
       ;;
   esac
   return 0
 }
 
 # **Internal** Main program.
+# shellcheck disable=2120,2154
 _main() {
-  build-docker-image $@
+  build-docker-image "$@"
```

Comment:
```
In ./components/bintray-publish/bin/publish-studio.sh line 145:
_main() {
^-- SC2120: _main references arguments, but none are ever passed.
```
Either `build-docker-image "$@"` should have the `"$@"` removed or

Reply:
I opted to remove the `$@` from `build-docker-image` as I think it's better to be explicit than implicit in passing arguments.